### PR TITLE
[IMP] l10n_be_hr_payroll: manage credit time during contract

### PR DESCRIPTION
**Pull request by jbm-odoo**
Description of the issue/feature this PR addresses:
In Belgium, we have the concept of time credit and parental leaves.
Globally the person keep full time rights and the initial contract
is always signed on a full time basis. But an avenant with the new
time schedule must be signed.

Desired behavior after PR is merged:
A new wizard on contract allows to manage credit time. We can specify
the start and stop date, the new calendar and the work time ratio.
The new wage will be compute.
After validation, the current contract will be duplicate once: for the time 
credit contract. The current contract will have end date at
the beginning of the time credit period.

id=1924200

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/resource/models/resource.py [hunk 1]
```diff
@@ -151,29 +151,28 @@ def default_get(self, fields):
         res = super(ResourceCalendar, self).default_get(fields)
         if not res.get('name') and res.get('company_id'):
             res['name'] = _('Working Hours of %s') % self.env['res.company'].browse(res['company_id']).name
+        if not res.get('attendance_ids'):
```
Comment (RomainLibert) [inline addons/resource/models/resource.py]: Could you explain why you had to move this inside the default_get?

File: addons/resource/views/resource_views.xml [hunk 2]
```diff
@@ -257,7 +257,7 @@
                     <group name="resource_details">
                         <group>
                             <field name="company_id" groups="base.group_multi_company"/>
-                            <field name="hours_per_day"/>
+                            <field name="hours_per_day" groups="base.group_no_one"/>
```
Comment (RomainLibert) [inline addons/resource/views/resource_views.xml]: Why is this changed to debug mode only?

Comment (jbm-odoo) [reply addons/resource/views/resource_views.xml]: It's Aline specification

File: addons/resource/models/resource.py (hunk 1 above)
Comment (jbm-odoo) [reply addons/resource/models/resource.py]: It's useful for resource in payroll, we override the default_get, inside we copy default attendance_ids to normal_attendance_ids and remove default attendance_ids to avoid to have default data 2 times

File: addons/resource/views/resource_views.xml (hunk 2 above)
Comment (Yenthe666) [reply addons/resource/views/resource_views.xml]: Are we sure to do that? 🤔

**Comment by tivisse** [comment]
@robodoo rebase-ff r+
